JamJar/AutoCompleteTextField/VideoCell.swift

```diff
@@ -18,8 +18,13 @@ class VideoCell: UITableViewCell {
     
     var video: Video!
     
-    func setup(video: Video) {
+    // Store the viewController that we'll push the profile to if the user
+    // button is clicked
+    var viewController: UIViewController!
```

Comment:
Please make sure that this two way reference does not prevent "deinit" from being called for the view controllers.
weak variables will be a solution if there is a problem.